test/built-ins/Date/UTC/nans.js

```diff
@@ -14,13 +14,16 @@ info: |
   7. If ms is supplied, let milli be ? ToNumber(ms); else let milli be 0.
   8. If y is not NaN and 0 ≤ ToInteger(y) ≤ 99, let yr be 1900+ToInteger(y);
      otherwise, let yr be y.
-  9. Return TimeClip(MakeDate(MakeDay(yr, m, dt), MakeTime(h, min, s, milli))). 
+  9. Return TimeClip(MakeDate(MakeDay(yr, m, dt), MakeTime(h, min, s, milli))).
 ---*/
 
-assert.sameValue(new Date(NaN, 0).getTime(), NaN, 'year');
-assert.sameValue(new Date(1970, NaN).getTime(), NaN, 'month');
-assert.sameValue(new Date(1970, 0, NaN).getTime(), NaN, 'date');
-assert.sameValue(new Date(1970, 0, 1, NaN).getTime(), NaN, 'hours');
-assert.sameValue(new Date(1970, 0, 1, 0, NaN).getTime(), NaN, 'minutes');
-assert.sameValue(new Date(1970, 0, 1, 0, 0, NaN).getTime(), NaN, 'seconds');
-assert.sameValue(new Date(1970, 0, 1, 0, 0, 0, NaN).getTime(), NaN, 'ms');
+assert.sameValue(Date.UTC(NaN, 0), NaN, 'year');
```

Comment:
as we are tests the new defined two forms of Date.UTC, with one or two arguments, can we __clone__ this assertion to use the single argument form?

```js
assert.sameValue(Date.UTC(NaN), NaN, 'year');
```